Doppler.Sap/Doppler.Sap.csproj

```diff
@@ -4,8 +4,14 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="Enums\CountryCodeEnum.cs" />
```

Comment:
Is necessary?

Reply:
yes, because the file was removed in the project

Reply:
if it is removed, why is necessary a line? 🤔

Reply:
It line was added automatically in the project when I removed the file

Reply:
Revisalo, no creo que esa línea sea correcta, tal vez removiste el archivo sin eliminarlo. Lo mismo para BusinessPartnerMapper.cs

Reply:
Ok, I will review it.

Reply:
Done